### PR TITLE
News week milestone3

### PR DESCRIPTION
**Pull request by fortuneonyeka**
Milestone3 with bootstrap

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -638,6 +638,407 @@ <h5 class="mw-100 mt-3">START YOUR DAY WITH OUR TOP 5 ARTICLES</h5>
                 </div>
                 </section>
                 </main>
+                 <!------------Milestone 3--------->
+            <div class="container mt-5">
+                <h5 class="mw-100 text-center border-bottom text-dark text-decoration-underline font-weight-bolder"><img class="p-2" src="img/favicon-32x32.png" alt="" />IN THE MAGAZINE</h5>
+                <div class="row">
```
Comment (christianotieno) [inline index.html:644]: ### Spacing
- [x] Let's be mindful of the spacings on the rest of the contest. They all should be having a uniform margin as the page need not be covering the whole view width of the screen bootstrap `contani3er` class can help us achieve this, do you mind implementing thin?

|Source Page|Clone Page|
|:--:|:--:|
|source|clone|

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Changes required ♻️ 
Hello @fortuneonyeka & @marcosmerida  👋🏾 

Congratulations on the hard work done in the third, milestone of this project.
Kindly note that for this final milestone, we need to ensure nothing is made up. Let's take care of the small suggested change below before we proceed.

## mandatory
- [x] Take note of the spacings required on the `navbar`, `footer` and the first section of the page.

### Happy coding 💻 
#### Christian Otieno (CR)

**Review by george-shammar — APPROVED** [review]
### **STATUS: APPROVED** :heavy_check_mark: :heavy_check_mark: 

**You've done a great job**. :ok_hand: :ok_hand: 

**Congratulations.** :+1: :+1: 

[OPTIONAL]

- [ ] **Further reduce the margin on both sides of the page. The margin is still wider than the original page. With that, your project would be almost flawless.**

**You may go ahead and merge this branch.**

**I wish you good luck with your future projects.**

**Comment by fortuneonyeka** [comment]
> ### **STATUS: APPROVED** ✔️ ✔️
> **You've done a great job**. 👌 👌
> 
> **Congratulations.** 👍 👍
> 
> [OPTIONAL]
> 
> * [ ]  **Further reduce the margin on both sides of the page. The margin is still wider than the original page. With that, your project would be almost flawless.**
> 
> **You may go ahead and merge this branch.**
> 
> **I wish you good luck with your future projects.**

Thank you very much, George

**Pull request merged**